frontend/src/index.css

```diff
@@ -80,11 +80,48 @@ select {
 }
 
 body {
-    background-color: #f7f8fc;
+    background-color: #e7e7e7;
```

Comment:
if this is supposed to be a theme color, please use `theme.uniformStyle.colors. ...`  instead

Reply:
Oh wait, sry, we are in the index.css file! That won't work, but maybe @IremToroslu we can have a look into this matter later as well :)

Reply:
Nice that you used the .css file right away!

Reply:
we also need to look over the .css file I guess I would like to discuss some points as well and sure, I am on it